plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceProviderDescriptor.java

```diff
@@ -174,9 +174,18 @@ public DataSourceProviderDescriptor(DataSourceProviderRegistry registry, IConfig
 
         // Load native clients
         {
-            for (IConfigurationElement nativeClientsElement : config.getChildren("nativeClients")) {
-                for (IConfigurationElement clientElement : nativeClientsElement.getChildren("client")) {
-                    this.nativeClients.add(new NativeClientDescriptor(clientElement));
+            IConfigurationElement[] nativeClients = config.getChildren("nativeClients");
+            if (ArrayUtils.isEmpty(nativeClients)) {
+                String inheritClients = config.getAttribute("inheritClients");
+                if (CommonUtils.isNotEmpty(inheritClients) && inheritClients.equals("true") && parentProvider != null) {
```

Comment:
Instead of an equality check for a boolean, you can use `CommonUtils#toBoolean`.